components/suggest/src/suggest.udl

```diff
@@ -40,6 +41,16 @@ interface Suggestion {
         sequence<u8>? icon,
         string full_keyword
     );
+    Amo(
+        string title,
+        string url,
+        string icon,
```

Comment:
```suggestion
        string icon_url,
```

suggestion: Ditto about `icon_url` here, to match our other `_url`-suffixed names for URLs.